<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="add_new_match_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Add new match</string>
   </property>
   <widget class="QComboBox" name="player1_combo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="player2_combo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="score1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>52</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="score2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>52</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
   </widget>
   <widget class="QComboBox" name="serve_player_combo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>78</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>194</width>
      <height>23</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2013</year>
      <month>7</month>
      <day>1</day>
     </date>
    </property>
    <property name="time">
     <time>
      <hour>8</hour>
      <minute>0</minute>
      <second>0</second>
     </time>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Player 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_new_match_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add match</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score Player 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Score Player 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Serve</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="view_match_table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
